README.md

```diff
@@ -105,8 +105,8 @@ The following Environment Variables must be set in your shell prior to running a
 - `ARM_CLIENT_SECRET`
 - `ARM_SUBSCRIPTION_ID`
 - `ARM_TENANT_ID`
-- `ARM_ENVIRONMENT`
-- `ARM_TEST_LOCATION`
+- `ARM_ENVIRONMENT`: Possible values are public, usgovernment, german, and china. Defaults to public.
+- `ARM_TEST_LOCATION`: For example, 'eastus', 'centralus', etc.
```

Comment:
Could we remove this change and pop it in to a 2nd pr?

Reply:
sure

Reply:
I'ved removed that